src/theme/material/templates/plugin-column-filter-date.tpl.html

```diff
@@ -0,0 +1,21 @@
+<ng-template key="plugin-column-filter-date.tpl.html"
+             let-$columnFilter
+             let-$plugin="plugin"
+             let-$vscroll="vscroll">
+
+	<mat-form-field class="q-grid-column-filter-input"
+	                *q-grid-date="let isValid='isValid'"
+					[ngClass]="{'mat-form-field-invalid': !isValid(input.value)}">
+					
+		<input q-grid-focus
+		       matInput
+		       #input
+		       [matDatepicker]="picker"
+		       [placeholder]="$columnFilter.expression.op | qGridText : 'fromCamelCase'"
```

Comment:
can we mark expression as private and not use in template? it forces user to know expression structure during development